README.md

```diff
@@ -26,6 +26,11 @@ $ pip3 install "odoo-analyse[graph]"
 
 `-s /path/to/data.json` .. Store the loaded modules in a file
 
+Or if you want to output it to `stdin`:
+
+`-s -` .. Outputs the loaded modules to stdin
```

Comment:
It's stdout twice. Could you also add the `--load - .. Load the modules from stdin` in the section above?

Reply:
Euhm, how do you mean it is `stdout` twice?
& about `--load`: hmm, I don't see to get this command honestly 🤔

Reply:
You output to stdout and not stdin as described in the PR.

`-l/--load` can load files created by `-s/--save`. This was initially done to add a cache. E.g. if you analyse an entire Odoo instance it takes quite long. But if you want to see different graphs afterwards you have to run it twice. This is the reason we implemented `-s` and `-l` in the first place. Analyse bigger repositores/Odoo installations and save it into a file. Afterwards you can create your statistic out of the file which is fast or share the analysed data.